pyproject.toml

```diff
@@ -154,7 +154,7 @@ volint = "astropy.io.votable.volint:main"
 wcslint = "astropy.wcs.wcslint:main"
 
 [build-system]
-requires = ["setuptools",
+requires = ["setuptools>=72.2.0",
```

Comment:
note that this is because setuptools 72.2.0 is needed to support extensions specified as `Path`s, see https://setuptools.pypa.io/en/latest/history.html#id35

Reply:
will revert this (see https://github.com/astropy/astropy/pull/16942)